setup.py

```diff
@@ -34,11 +31,10 @@ def finalize_options(self):
         pass
 
     def run(self):
-        os.system('python setup.py sdist bdist_wheel')
-
-        sdist = 'dist/icecream-%s.tar.gz' % meta['__version__']
-        wheel = 'dist/icecream-%s-py2.py3-none-any.whl' % meta['__version__']
-        rc = os.system('twine upload "%s" "%s"' % (sdist, wheel))
+        dist_dir = pjoin(this_dir, "dist")
+        os.system(sys.executable + " -m build -nwxs " + this_dir)
+        files = glob(pjoin(dist_dir, "*.whl")) + glob(pjoin(dist_dir, "*.whl"))
```

Comment:
```suggestion
        files = glob(pjoin(dist_dir, "*.whl")) + glob(pjoin(dist_dir, "*.tar.gz"))
```

Is this what you meant?

Reply:
Yeah. Thanks. Fixed.